_integration-tests/tests/vault/main.go

```diff
@@ -0,0 +1,71 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package vault
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	testvault "github.com/testcontainers/testcontainers-go/modules/vault"
+)
+
+type TestCase struct {
```

Comment:
This test case is not executed... it doesn't implement the interface properly as it's missing expected traces...